specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2020-02-07-preview/examples/SapMonitors_Create.json

```diff
@@ -0,0 +1,80 @@
+{
+  "parameters": {
+    "api-version": "2020-02-07-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "sapMonitorName": "mySapMonitor",
+    "sapMonitorParameter": {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.HanaOnAzure/sapMonitors/mySapMonitor",
+      "name": "mySapMonitor",
+      "type": "Microsoft.HanaOnAzure/sapMonitors",
+      "location": "westus",
+      "tags": {
+        "key": "value"
+      },
+      "enableCustomerAnalytics": true,
+      "logAnalyticsWorkspaceArmID": "",
+      "logAnalyticsWorkspaceID": "",
+      "logAnalyticsWorkspaceSharedKey": "",
+      "monitorProviders": [
```

Comment:
Should be `providerInstances` to be consistent

Reply:
currently the code is monitorProviders: https://dev.azure.com/msazure/One/_git/Compute-AzLinux-HanaRp?path=%2Fsrc%2Fhanarp%2Fapi%2Fapiversions%2Fv20200207%2Ftypes.go&version=GBmaster&line=22&lineEnd=22&lineStartColumn=64&lineEndColumn=80&lineStyle=plain

Do you want me to change it there too?